code/aes/Makefile

```diff
@@ -0,0 +1,35 @@
+HACL_HOME=$(realpath ../..)
```

Comment:
does this work? has this been tested a little bit?

Reply:
the whole local Makefile, I mean

Reply:
It does work, we used the local Makefile to test the implementation but since we added support for aes test in tests/Makefile we can remove this one.

Reply:
keep the Makefile, I just wanted to make sure we didn't check in a Makefile that hadn't been tested before